packages/commands/src/lib/integrations.ts

```diff
@@ -167,4 +176,54 @@ export default defineConfig({
 			}
 		},
 	},
+  "tauri-v1.x": {
+    installs: ["@tauri-apps/cli"],
+    postInstall: async () => {
+      try {
+        const packageJsonString = await readFile("package.json", "utf8");
+        const packageJson = JSON.parse(packageJsonString);
+        if (!packageJson.scripts) { packageJson.scripts = {}; }
+        packageJson.scripts.tauri = "tauri";
+        await writeFile(
+          "package.json",
+          JSON.stringify(packageJson, null, /^(\t|\s+)/m.exec(packageJsonString)?.[0] || 2) + "\n",
+          "utf8"
+        );
+        await flushQueue();
+        const name = packageJson.name;
+        const pM = detectPackageManager();
+        await $`${getRunnerCommand(pM)} tauri init --ci -A ${name} -W ${name} -D ../dist -P http://localhost:3000`;
+        p.outro(`Make sure you have installed all prerequisites: https://tauri.app/v1/guides/getting-started/prerequisites
+
+    Start tauri development with ${color.bold(pM.name)} ${color.bold(pM.runScriptCommand("tauri dev"))}`);
+      } catch (err) {
+        console.error(err);
+      }
+    },
+  },
+  "tauri-v2.x": {
+    installs: ["@tauri-apps/cli@next"],
+    postInstall: async () => {
+      try {
+        const packageJsonString = await readFile("package.json", "utf8");
+        const packageJson = JSON.parse(packageJsonString);
+        if (!packageJson.scripts) { packageJson.scripts = {}; }
+        packageJson.scripts.tauri = "tauri";
+        await writeFile(
+          "package.json",
+          JSON.stringify(packageJson, null, /^(\t|\s+)/m.exec(packageJsonString)?.[0] || 2) + "\n",
+          "utf8"
+        );
+        await flushQueue();
+        const name = packageJson.name;
+        const pM = detectPackageManager();
+        await $`${getRunnerCommand(pM)} tauri init --ci -A ${name} -W ${name} -D ../dist -P http://localhost:3000`;
+        p.outro(`Make sure you have installed all prerequisites: https://v2.tauri.app/start/prerequisites/
```

Comment:
Would `p.log.info` or `p.note` work here?

Reply:
I'll try it later.